modules/geolocation/unittests.py

```diff
@@ -194,8 +194,68 @@ def test_point_set_2(self):
         actual = locator.calculate_pixel_to_geo_mapping()
         # Test
         np.testing.assert_almost_equal(actual, expected)
+    
+class TestMapLocationFromPixel(unittest.TestCase):
+    """
+    Tests Geolocation.map_location_from_pixel()
+    """
+    def setUp(self):
+        
+        self.locator = geolocation.Geolocation()
+
+        return
+    
+    def test_identity_transformation_matrix(self):
+
+        # Setup
+        identityTransformationMatrix = np.ones(shape=(3,3))
```

Comment:
This is not an identity matrix. This is a matrix filled with 1s. Consider `numpy.eye()`.

Does this test pass? I'm looking at it and my current thought is no.

Reply:
I meant to say a transformation matrix filled with ones, instead of identity matrix.

Changed the name of the function and the associated variable name. Test Passed.